resources/assets/js/videos/components/videoScreen/videoPlayback.vue

```diff
@@ -153,7 +155,23 @@ export default {
         handleSeeked() {
             this.renderVideo(true);
         },
-        // 3 next methods are a workaround to get previous and next frames, adapted from here: https://github.com/angrycoding/requestVideoFrameCallback-prev-next/tree/main
+        // 5 next methods are a workaround to get previous and next frames, adapted from here: https://github.com/angrycoding/requestVideoFrameCallback-prev-next/tree/main
+        async emitPreviousFrame() {
+            if(this.video.currentTime == 0 || this.seekingFrame) return;
```

Comment:
I can replace `this.seekingFrame` with `this.seeking` here and see no difference.